vrx_gazebo/models/crocodile_static/model.sdf

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <model name="crocodile_buoy">
+    <link name="link">
+      <inertial>
+        <pose>0 0 -2 0 0 0</pose>
+        <!-- Mass determined based on buoyancy geometry so object 
+             floats at mid-point of sphere -->
+        <mass>20</mass>
+        <!-- Mark only - based on cylinder -->
+        <inertia>
+          <ixx>1.0</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>1.0</iyy>
+          <iyz>0</iyz>
+          <izz>0.1</izz>
+        </inertia>
+      </inertial>
+      <collision name="crocodile_collision">
+        <pose>0 0 0.06 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.80 0.40 0.15</size>
+          </box>
+        </geometry>
+      </collision>
+      <visual name="crocodile_visual">
+        <geometry>
+	        <mesh><uri>model://crocodile_buoy/meshes/crocodile.dae</uri></mesh>
```

Comment:
This is pointing to the non-static version, which is totally fine. I think you can remove all the `.dae`, `.png` and `.material` from the static versions. It should be just the `model.sdf` and the `model.config` reusing all the resources from the non-static version for example.

Reply:
Removed files and double-checked for correctness